core/domain/search_services.py

```diff
@@ -39,6 +43,9 @@
 # Name for the collection search index.
 SEARCH_INDEX_COLLECTIONS: Final = 'collections'
 
+# Name for the blog post search index.
```

Comment:
@Rijuta-s -- when you merge develop into this branch per Kevin's instructions, would you please also add to this comment, and the two above, the following text?

"NOTE TO DEVELOPERS: If you change any of these index names or add any new indexes, please contact Sean  to update permissions on the ElasticSearch production servers, otherwise search operations will fail in production. Please do this before merging the PR. Thanks!"

Reply:
Done!